VMEncryption/main/handle.py

```diff
@@ -612,9 +617,12 @@ def enable():
         logger.log('Data Disks Status: {0}'.format(encryption_status['data']))
         logger.log('OS Disk Status: {0}'.format(encryption_status['os']))
 
+        encryption_operation = public_settings.get(CommonVariables.EncryptionEncryptionOperationKey)
+
         # run fatal prechecks, report error if exceptions are caught
         try:
-            cutil.precheck_for_fatal_failures(public_settings, encryption_status, DistroPatcher, existing_volume_type)
+            if not encryption_operation == CommonVariables.Migrate:
+                cutil.precheck_for_fatal_failures(public_settings, encryption_status, DistroPatcher, existing_volume_type)
```

Comment:
are there any prechecks we want to include even in migrate case?  for example key vault syntax validation